crates/wash-cli/tests/wash.rs

```diff
@@ -38,4 +39,25 @@ fn integration_help_subcommand_check() {
     assert!(output.contains("drain"));
     assert!(output.contains("keys"));
     assert!(output.contains("claims"));
+    Ok(())
+}
+
+/// Ensure `wash -h` works
+#[test]
+fn integration_help_short_works() -> Result<()> {
+    wash()
+        .args(["-h"])
+        .output()
```

Comment:
Shouldn't there be a few trivial output.contains here?

Reply:
This would help more on the other new test -- `-h` is the short form of `--help` so it *should* be fine -- if another short form was added, this would panic (which is why this test was added -- @brooksmtownsend ran into that panic after a recent change).

what might make sense is some includes for `--help-markdown`, so adding that!